paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/BillingDetailsHelpers.kt

```diff
@@ -143,8 +143,10 @@ internal object BillingDetailsHelpers {
 
             countryElement?.controller?.rawFieldValue ?: emptyFlow()
         }.collect {
-            it?.let {
-                phoneNumberElement?.controller?.countryDropdownController?.onRawValueChange(it)
+            if (phoneNumberElement?.controller?.getLocalNumber().isNullOrBlank()) {
```

Comment:
Nit: We could solve this with a `filterNotNull()` in the Flow chain.

Reply:
I think the tests surrounding this are pretty decent now. I don't mind improving things!

If you add `.filterNotNull()` to line 145 you shouldn't need the let!

Reply:
Ah! I was thinking of the wrong thing